src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/ResourceServerConfiguration.kt

```diff
@@ -30,6 +30,7 @@ class ResourceServerConfiguration {
           "/v3/api-docs/**",
           "/swagger-ui/**",
           "/swagger-ui.html",
+          "/openapi/**",
```

Comment:
Allow anything from the `/openapi` path (which maps to our `resources/static/openapi` folder)